forge-gui/res/quest/world/worlds.txt

```diff
@@ -37,4 +37,5 @@ Name:Kaladesh|Sets:AER, KLD, MPS, KLR
m;Wretched Gryff;Maelstrom Colossus;Spare Supplies;Vulshok Morningstar;Pristine Talisman;Sickleslicer;Trailblazer's Torch;Thundersteel Colossus;Glacial Floodplain;Razortide Bridge;Skybridge Towers;Ice Tunnel;Mistvault Bridge;Waterfront District;Drossforge Bridge;Sulfurous Mire;Tramway Station;Highland Forest;Racers' Ring;Slagwoods Bridge;Arctic Treeline;Botanical Plaza;Thornglint Bridge;Silverquill Campus;Snowfield Sinkhole;Prismari Campus;Volatile Fjord;Witherbloom Campus;Woodland Chasm;Alpine Meadow;Lorehold Campus;Quandrix Campus;Rimewood Falls;Barren Moor;Brokers Hideout;Cabaretti Courtyard;Cryptic Spires;Desert;Forgotten Cave;Lonely Sandbar;Maestros Theater;Obscura Storefront;Riveteers Overlook;Secluded Steppe;Thriving Bluff;Thriving Grove;Thriving Heath;Thriving Isle;Thriving Moor;Tranquil Thicket;Deftblade Elite;Miner's Guidewing;Ainok Bond-Kin;Phantom Nomad;Topan Freeblade;Holy Cow;Blessed Hippogriff;Oltec Cloud Guard;Seraph of Dawn;Alabaster Host Intercessor;Imperial Oath;Pegasus Guardian;Angelic Intervention;Dauntless Unity;Unbounded Potential;Borrowed Grace;Recommission;Greatsword of Tyr;Cooped Up;Petrify;Realmbreaker's Grasp;Bound in Gold;Callous Dismissal;Merfolk Looter;Zephyr Winder;Geyser Drake;Mistral Singer;Mocking Sprite;Preening Champion;Augury Raven;Cruel Witness;Young Blue Dragon;Sword Coast Serpent;Displace;Quick Study;Witching Well;Bind the Monster;Weakstone's Subjugation;Witness Protection;Nezumi Linkbreaker;Persistent Specimen;Unwilling Ingredient;Gixian Infiltrator;Guildsworn Prowler;Legion Vanguard;Mire Triton;Mold Folk;Dunland Crebain;Voracious Vermin;Diregraf Horde;Loathsome Curator;Feign Death;Lash of Malice;Undying Malice;Village Rites;Candy Grapple;Rowan's Grim Search;Feed the Swarm;Ransack the Lab;Drown in Sorrow;Arms of Hadar;Tithing Blade;Tortured Existence;Goblin Motivator;Harried Spearguard;Reckless Lackey;Barbed Batterfist;Hobblefiend;Rally at the Hornburg;Ratcatcher Trainee;Red Herring;Volatile Wanderglyph;Yavimaya Steelcrusher;Spelleater Wolverine;Spellgorger Weird;Tuskeri Firewalker;Chimney Rabble;Wildfire Elemental;Improvised Club;Ambitious Assault;Gnawing Crescendo;Sulfurous Blast;Wrangle;Wrenn's Resolve;Raid Bombardment;Bayou Groff;Gnarlid Colony;Gnoll Hunter;Jade Avenger;Thornweald Archer;Voracious Varmint;Wary Thespian;Werebear;Contagious Vorrac;Deepwood Denizen;Phantom Tiger;Tuskguard Captain;Bloom Hulk;Jade Guardian;Urban Daggertooth;Voracious Typhon;Silverback Shaman;Greater Tanuki;Dread Linnorm;Massive Might;Tamiyo's Safekeeping;Grapple with the Past;Master's Rebuke;Prizefight;Clear Shot;Edge of Autumn;You Meet in a Tavern;Wild Growth;Silver Drake;Body Dropper;Goblin Anarchomancer;Goblin Electromancer;Infuse with Vitality;Dog Walker;War Flare;Iron Apprentice;Filigree Familiar;Skyscanner;Juggernaut;Candy Trail;Leonin Bola;Prophetic Prism;Star Compass;Jagged Barrens;Bristling Backwoods;Eroded Canyon;Abraded Bluffs;Basilisk Gate;Conduit Pylons;Crystal Grotto;Quicksand;Shire Terrace;Tocasia's Dig Site
```

Comment:
This.. seriously explodes this file. Can we look into doing this a different way? Maybe having worlds be individual files, and have them have sections kinda like an Edition? Sets, Cards, etc.